examples/minimal-mdns/advertiser.cpp

```diff
@@ -289,7 +289,7 @@ int main(int argc, char ** args)
                 .EnableIpV4(gOptions.enableIpV4)
                 .SetPort(CHIP_PORT)
                 .SetMac(chip::ByteSpan(gOptions.mac, 6))
-                .SetPeerId(PeerId().SetFabricId(gOptions.fabricId).SetNodeId(gOptions.nodeId)));
+                .SetPeerId(PeerId().SetCompressedFabricId(gOptions.fabricId).SetNodeId(gOptions.nodeId)));
```

Comment:
In this example, setting direct compressed fabric ID seems odd and doesn't match what I would expect a tool user to do.

Reply:
We'll need to provide the root pub key to the tool to compute the compressed fabric ID. That could be done, but need a clever way of grabbing the correct pub key, and be able to pass it to the command line tool.

I can file an issue and tag it with this change..

Reply:
https://github.com/project-chip/connectedhomeip/issues/9193

Reply:
Suggest that commissioning context be configurable in tools so that for dev testing, you can "name"/"alias" a fabric to its combination of <root cert, fabric id> and also configure the dev ICA (if any) for a given alias for exemplary credential issuance. This would go a very long way in help support multi-fabric testing.